setup.py

```diff
@@ -33,7 +33,7 @@
     install_requires=[
         'cloudify-plugins-common>=3.4',
         'requests==2.20.0',
-        'pyyaml==3.10',
+        'pyyaml>=4.2b1'
```

Comment:
@EarthmanT do not you think this will conflict with the pyyaml versions pinned by cloudify ? since cloudify common pinned `3.10`

Reply:
@mabuaisha yeah I also see that we are already using `safe_load`.